doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -20,17 +20,17 @@ and its private keyring.
 
 It is also possible to deploy BlueStore across two additional devices:
 
-* A *WAL device* can be used for BlueStore's internal journal or
-  write-ahead log.  It is identified by the ``block.wal`` symlink in
-  the data directory.  It is only useful to use a WAL device if the
-  device is faster than the primary device (e.g., when it is on an SSD
-  and the primary device is an HDD).
-* A *DB device* can be used for storing BlueStore's internal metadata.
-  BlueStore (or rather, the embedded RocksDB) will put as much
-  metadata as it can on the DB device to improve performance.  If the
-  DB device fills up, metadata will spill back onto the primary device
-  (where it would have been otherwise).  Again, it is only helpful to
-  provision a DB device if it is faster than the primary device.
+* A *WAL device* (identified as ``block.wal`` in the data directory) can be
```

Comment:
Is there any reason to use a wal partition instead of a db partition if you only have 2 devices (even if only a wal will fit on the faster one)?

That would simplify this section a bit - we could mention separate wal partitions at the end, only for a case where you have an even faster 3rd device

Reply:
I don't think there is a reason, no. I agree pushing them to the end, since we are concentrating in `block.db` here